src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.service;
+
+import static core.basesyntax.db.Storage.people;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static final User USER_VALID = new User("Badcannon", "password1", 39);
+    private static final User USER_WITH_SHORT_PASSWORD = new User("Sam", "12345", 39);
+    private static final User USER_WITH_EMPTY_LOGIN = new User("", "123456", 39);
+    private static final User USER_WITH_NULL_LOGIN = new User(null, "123456", 39);
+    private static final User USER_AGE_LESS_THAN_REQUIRED = new User("John", "password4", 15);
```

Comment:
We should create test objects as constants only if we use them several times in code, I think it is better to make these variables local